packages/app/client/src/reduxStructures/profile/dataManipulation.js

```diff
@@ -8,3 +8,40 @@ export const onSaveInfos = (state, payload) => {
     badges: userBadges
   }
 }
+// TODO: MAKE THE FUNCTIONS BELOW AN UTILITY BECAUSE THEIR LOGIC HAVE BEEN USED ON PROJECT REDUCER AS WELL
```

Comment:
Not keen on todos in code we should use zube to track things that need to be done. You can link to the line in the code there

Reply:
will do!